src/main/java/net/mcreator/ui/dialogs/preferences/PluginsPanel.java

```diff
@@ -116,21 +116,21 @@ public Component getListCellRendererComponent(JList<? extends Plugin> list, Plug
 			ComponentUtils.deriveFont(this, 12);
 
 			if ((value.getInfo().getAuthor() != null) && (value.getInfo().getName() != null) && (value.getInfo()
-					.getCredits().equals("None"))) {
-				setText("<html>" + value.getInfo().getName() + "<br><small>ID: " + value.getID() + ", version: " + value
-						.getPluginVersion() + ", author: " + value.getInfo().getAuthor() + ", loaded: " + (value
-						.isLoaded() ? "<html><font color=#a7ed1a>yes</font>" : "<html><font color=#f24122>no</font>"));
+					.getCredits().equals(L10N.t("dialog.preferences.plugins_none")))) {

```

Comment:
None will be returned by the plugin loader so this can't be translated. I think the best bet here is to leave the plugin list renderer untranslated.